dts/bindings/gpio/cypress,cy8c95xx-gpio.yaml

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2021 Synopsys
+# SPDX-License-Identifier: Apache-2.0
+
+description: Cypress cy8c95xx GPIO PORT node
+
+compatible: "cypress,cy8c95xx"
+
+include: ["gpio-controller.yaml", "base.yaml"]
+
+properties:
+    reg:
+      required: true
+
+    label:
+      required: true
+
+    "#gpio-cells":
+      const: 2
+
+    ngpios:
+      required: true
+      const: 16
+
+    init-dir:
+      type: int
+      required: false
+      default: 0
+      description: |
+        Bit mask identifying pins initial dir.
+
+    init-data:
+      type: int
+      required: false
+      default: 0
+      description: |
+        Bit mask identifying pins initial data.
```

Comment:
This should be implemented using the so called [gpio hogging mechanism](https://github.com/torvalds/linux/blob/89698becf06d341a700913c3d89ce2a914af69a2/Documentation/devicetree/bindings/gpio/gpio.txt#L185) and respective properties. We don't support it currently. Unless you really need it I propose to remove above properties for now.

Reply:
removed.